sdk/agrifood/Azure.Verticals.AgriFood.Farming/tests/Generated/Samples/Samples_Boundaries.cs

```diff
@@ -6,6 +6,7 @@
 #nullable disable
 
 using System;
+using System.Collections.Generic;
```

Comment:
Do we need this library? There is no other code changes, and previous code can compile without this, so it should not need this.